### PR TITLE
[Assembly v1] Phone

### PR DESCRIPTION
**Pull request by katydecorah**
This PR merge into #481 and updates the Phone component to Assembly v1.

- [x] Run codemod and look out for additional changes
- [x] Check docs-prose.css for any component-specific classes that should be updated or removed, including changes to the color scale.
- [x] Visually check test cases app for component

Note: the test cases app uses the Video component, which hasn't been updated yet (you may see a mis-aligned video play button).

Current | Proposed
---|---
screencapture-192-168-7-120-9966-Phone-2021-09-09-20_17_03 | screencapture-192-168-7-120-9966-Phone-2021-09-09-20_15_40

## QA checklist

- [x] No errors logged to console.
- [x] Accessibility score in [Chrome DevTools Audit/Lighthouse](https://developers.google.com/web/tools/lighthouse#devtools) is 100% with Lighthouse version: `#.#.#`.
- [x] Component is accessible at mobile-size viewport.
- [x] Component is accessible at tablet-size viewport.
- [x] Component is accessible at laptop-size viewport.
- [x] Component is accessible at big-monitor-size viewport.

Open the test cases app locally on:

- [x] Chrome, no errors logged to console.
- [x] Firefox, no errors logged to console.
- [x] Safari, no errors logged to console.
- [x] Edge, no errors logged to console.
- [x] Mobile Safari, no errors logged to console.
- [x] Android Chrome, no errors logged to console.

## Before merge

- [ ] Add entry to CHANGELOG.md to describe changes.
- [ ] If updating dependencies or creating a release, run `npx browserslist@latest --update-db` to update the [browser data](https://github.com/browserslist/browserslist#browsers-data-updating) and commit any changes to package-lock.json.

### CODE REVIEW

File: src/components/phone/__tests__/__snapshots__/phone.test.js.snap [hunk 1]
```diff
@@ -31,26 +31,26 @@ exports[`phone Android landscape renders as expected 1`] = `
...
         }
       }
     >
       <div
-        className="bg-lighten10 round-full flex-child h60 w6"
+        className="bg-gray-deep round-full h60 w6"
       />
       <div
-        className="bg-lighten10 round-full w6 h6 flex-child absolute"
+        className="bg-gray-deep round-full w6 h6 absolute"
```
Comment (katydecorah) [inline src/components/phone/__tests__/__snapshots__/phone.test.js.snap:53]: 💡 I thought `bg-gray-deep` provided a more interesting contrast here.

File: src/components/phone/phone.js [hunk 2]
```diff
@@ -11,64 +11,58 @@ class Phone extends React.PureComponent {
     const portrait = props.mode === 'portrait';
     const landscape = props.mode === 'landscape';
 
+    const frameHex = '#0e2127'; // bg-gray-dark
```
Comment (katydecorah) [inline src/components/phone/phone.js:14]: 💡 I made this a variable so it's easier to update in the future and documented.

**Comment by malwoodsantoro** [comment]
@katydecorah still trying to figure out the root cause, but I am seeing that the videos are not aligning with the phone frames: 

image

image

They align as expected when viewing the test cases on `assembly-v1-update`:

image

image

**Comment by katydecorah** [comment]
@malwoodsantoro I had trouble with the test cases app caching Assembly (and drove me a little batty). Have you tried a hard refresh (shift + refresh)?

**Comment by malwoodsantoro** [comment]
@katydecorah thanks for the tip - I just did a lot of hard refreshing and I think we are back in action!

image

I'll continue reviewing this PR today 👍

**Review by malwoodsantoro — APPROVED** [review]
🐩

**Pull request merged**